src/Resources/translations/messages.en.yml

```diff
@@ -7,6 +7,24 @@ mango_sylius:
         message: 'Message'
         success: 'The message was sent with success'
         error: 'An error occurred when sending the message'
+        time: 'Send at %Time%'
```

Comment:
Always use a lowercase letter in variables

Reply:
DONE